enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -189,22 +201,22 @@ def _create_redemption(self, learner_id, content_key, **kwargs):
             learner_id=learner_id,
             content_key=content_key,
         )
-        transaction = self.create_transaction(
+        ledger_transaction = self.create_transaction(
             idempotency_key,
             quantity * -1,
             transaction_metadata,
         )
 
         try:
-            reference_id = self.enrollment_client.enroll(learner_id, content_key, transaction)
-            self.commit_transaction(transaction, reference_id)
+            reference_id = self.enrollment_client.enroll(learner_id, content_key, ledger_transaction)
+            self.commit_transaction(ledger_transaction, reference_id)
         except Exception as exc:
-            self.rollback_transaction(transaction)
+            self.rollback_transaction(ledger_transaction)
             raise exc
 
-        return transaction
+        return ledger_transaction
 
-    def _create_request(self, learner_id, content_key):
+    def _create_request(self, learner_id, content_key, **kwargs):
```

Comment:
FWIW, I don't think any of this request stuff belongs on the subsidy level - it should be up to the subsidy access policy to query the `subsidy_request` app for any existing requests.  What we'll want in the subsidy app is a consumer of siganls about requests/approval that will create a transaction/enrollment.